flocker/control/test/test_protocol.py

```diff
@@ -679,26 +729,60 @@ def test_configuration_change(self):
         """
         agent = FakeAgent()
         client = AgentAMP(Clock(), agent)
-        service = build_control_amp_service(self)
+        service_clock = Clock()
+        service = build_control_amp_service(self, service_clock)
         service.startService()
         server = LoopbackAMPClient(client.locator)
         service.connected(server)
 
         service.configuration_service.save(TEST_DEPLOYMENT)
+        service_clock.advance(10.0)
```

Comment:
:question: maybe use `BROADCAST_DELAY + 1` here too and elsewhere in the tests....or add an explanatory comment.